extensionsGallery-insider.json

```diff
@@ -3358,6 +3358,63 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "70",
+					"extensionName": "sql-database-projects",
+					"displayName": "SQL Database Projects",
+					"shortDescription": "The SQL Database Projects extension for Azure Data Studio allows users to develop and deploy database schemas.",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "0.1.0",
+							"lastUpdated": "06/24/2020",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/sql-database-projects/sql-database-projects-0.1.0.vsix"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/sql-database-projects/images/extension.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/sql-database-projects/README.md"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Code.Manifest",
+									"source": "https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/sql-database-projects/package.json"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.License",
+									"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt"
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ">=1.20.0"
```

Comment:
this property is not for version requirement, it is for extension dependencies, please leave the value as blank string.

Reply:
Whoops, that was some copy-pasta

Reply:
mssql actually doesn't need to be specified since it is built-in. you only need to specify: microsoft.schema-compare. in the future if you do need to specify multiple extensions, they should be separated by comma

Reply:
make sure to test the scenario that the schema-compare extension is not installed and the expected behavior is the extension will be installed together with schema-compare.